src/core/config.h

```diff
@@ -1,7 +1,7 @@
 #ifndef CORE_CONFIG_H
 #define CORE_CONFIG_H
 
-#define CONFIG_STRING_VALUE_MAX 65
+#define CONFIG_STRING_VALUE_MAX 66
```

Comment:
This can stay at `64` actually, the value dictates the max size of the string and not the number of strings.